docs/user/openstack/install_upi.md

```diff
@@ -269,6 +270,19 @@ api.openshift.example.com.    A 203.0.113.23
 
 They will need to be available to your developers, end users as well as the OpenShift installer process later in this guide.
 
+## Create network, API and ingress ports
+
+Please note, that the `dualstack-inventory.yaml` and `inventory.yaml` files will have the values for the API and Ingress VIPs overwritten. Run the following playbook to create necessary resources:
```

Comment:
Looks like you forgot to remove this reference to the `dualstack-inventory.yaml` that is now gone.